dev/assign_cherry_picked_prs_with_milestone.py

```diff
@@ -276,9 +275,7 @@ def assign_prs(
         console.print("\n[yellow]Implying --skip-assigned as summary report is enabled[/]\n")
         skip_assigned = True
     milestone = repo.get_milestone(milestone_number)
-    count_prs = len(prs)
-    if limit_pr_count:
-        count_prs = limit_pr_count
+    count_prs = limit_pr_count if limit_pr_count else None
```

Comment:
I think the `count_prs` variable is always the same as `limit_pr_count` (except for `0` but that value doesn’t make practical sense anyway)

Reply:
`limit_pr_count` can be `None` and we want to print the `count_prs` anyway, so I changed it back to real `len(prs)` if `limit_pr_count is None`.